statefun-flink/statefun-flink-harness/src/main/java/org/apache/flink/statefun/flink/harness/Harness.java

```diff
@@ -126,6 +129,14 @@ public Harness withSavepointLocation(String savepointLocation) {
   }
 
   public void start() throws Exception {
+    try {
+      startAsync().getJobExecutionResult().get();
+    } catch (ExecutionException ee) {
+      ExceptionUtils.rethrowException(ExceptionUtils.stripExecutionException(ee));
+    }
+  }
+
+  public JobClient startAsync() throws Exception {
```

Comment:
I'm not sure that I'd like to expose the `JobClient` as a public method on the `Harness`.
How about we think about it as a followup PR?

Reply:
Makes sense.